beacon-chain/rpc/beacon/assignments_test.go

```diff
@@ -393,3 +393,152 @@ func TestServer_ListAssignments_CanFilterPubkeysIndices_WithPagination(t *testin
 
 	assert.DeepEqual(t, wantedRes, res, "Did not receive wanted assignments")
 }
+
```

Comment:
Same to here

Reply:
We've refactor tests (new: `TestServer_MinimalConsensusSuite`)